src/__tests__/components.test.js

```diff
@@ -0,0 +1,180 @@
+// @flow
+import path from 'path'
+import pluginTester from 'babel-plugin-tester'
+import plugin from '../'
+
+const filename = path.resolve(__dirname, '..', '__fixtures__', 'messages.js')
+
+const defaultTest = {
+  title: 'default',
+  code: `
+import { FormattedMessage } from 'react-intl'
+
+console.log(<FormattedMessage defaultMessage="hello" />);
```

Comment:
Why need `console.log` ?

Reply:
It doesn't. I wanted to use the smallest possible valid code sample.i could just have the component by itself...It just felt like it would be floating...And not a typical use case 😄 happy to remove since I suppose Babel doesn't care

Reply:
I tried to remove this, and in fact it does throw an error when `<FormattedMessage>` is just in line on it's own. So, I've left the log to give it some structure.

Open to other ideas like a function:

```
const Component = () => <FormattedMessage />
```

or something?